src/docs/Components/Form/Checkbox/index.mdx

```diff
@@ -9,10 +9,74 @@ status:
 
 > Checkboxes are used to **select one or multiple options in a list**. They usually find their place in forms and are also used to accept some mentions.
 
-## Preview
-
 <Preview path="intro" nude />
 
+## Checkbox item
+
+The checkbox item is a combinaison of an input and a label.
+
+<Highlight theme="warning">
+
+  A checkbox can't live without an associated label.
```

Comment:
A checkbox can't be used without an associated label.

Reply:
fixed